packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -153,9 +183,21 @@ const helpers = {
           httpContent,
           Option.fold(
             () => {
-              logger.warn(`Unable to find a content for ${mediaTypes}`);
-              return Either.left<Error, IHttpNegotiationResult>(
-                ProblemJsonError.fromTemplate(NOT_ACCEPTABLE, `Unable to find content for ${mediaTypes}`),
+              return pipe(
+                optionallyGetPayloadlessResponse(response, headers || [], mediaTypes),
```

Comment:
We should come up with a better function name, this one is kind of awkward.

Reply:
@XVincentX, I'm all ears :)